src/servers/trigger.ts

```diff
@@ -95,7 +95,7 @@ export async function onchainTransactionEventHandler(tx) {
       return
     }
 
-    const user = await User.findOne({ _id: userId })
+    const user = await User.findOne({ _id: userId }, { lastIPs: 0, lastConnection: 0 })
```

Comment:
Can you explain why this change is needed?

Reply:
it's a performance optimization. ips array can be quite large currently because there is no pruning, and it's used by no application expect the one at the context. this would dramatically reduce the need of sending data to the wire between nodejs and mongodb for frequent users